This patch disables the pyne submodule check that occurs during the 
configuration step. pyne is not needed when BUILD_UWUW is disabled.    
author: Luke Labrie-Cleary
diff --unified --recursive --text DAGMC.orig/CMakeLists.txt DAGMC.new/CMakeLists.txt
--- DAGMC.orig/CMakeLists.txt	2024-07-10 10:38:54.657792978 +0200
+++ DAGMC.new/CMakeLists.txt	2024-07-10 10:40:10.547794197 +0200
@@ -42,9 +42,11 @@
 endif()
 
 # Check to see if submodules exist (by checking one)
-if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/src/pyne/pyne/readme.rst")
-  message(FATAL_ERROR "The git submodules were not downloaded! GIT_SUBMODULE was \
-    turned off or failed. Please update submodules and try again.")
+if(BUILD_UWUW)
+  if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/src/pyne/pyne/readme.rst")
+    message(FATAL_ERROR "The git submodules were not downloaded! GIT_SUBMODULE was \
+      turned off or failed. Please update submodules and try again.")
+  endif()
 endif()
 
 # Make the scripts in the "cmake" directory available to CMake
